packages/components/src/components/carousel/carousel.tsx

```diff
@@ -331,15 +331,15 @@ export class Carousel {
               <PrefixedTagNames.pButtonPure
                 {...btnProps}
                 icon="arrow-left"
-                ref={(ref) => (this.btnPrev = ref)}
+                ref={(ref: ButtonPure) => (this.btnPrev = ref)}
```

Comment:
I guess this should be `HTMLPButtonPureElement`. Also at the declaration of the private member

Reply:
<img width="1111" alt="image" src="https://github.com/user-attachments/assets/5e441756-be4a-421b-b97d-a4e802393635">